### PR TITLE
Add `isRepoNetworkGraph` and `isRepoForksList`

### PR DESCRIPTION
**Pull request by payne911**
For https://github.com/sindresorhus/refined-github/pull/3955

### CODE REVIEW

**Comment by payne911** [comment]
image

No clue what is the problem. :(

File: index.ts [hunk 1]
```diff
@@ -386,6 +386,16 @@ collect.set('isFileFinder', [
 	'https://github.com/sindresorhus/refined-github/find/master',
 ]);
 
+export const isForksList = (url: URL | HTMLAnchorElement | Location = location): boolean => getRepo(url)?.path === 'network/members';
+collect.set('isForksList', [
+	'https://github.com/sindresorhus/refined-github/network/members',
+]);
+
+export const isNetwork = (url: URL | HTMLAnchorElement | Location = location): boolean => getRepo(url)?.path === 'network';
+collect.set('isNetwork', [
```
Comment (fregante) [inline index.ts:395]:
```suggestion
export const isRepoNetworkGraph = (url: URL | HTMLAnchorElement | Location = location): boolean => getRepo(url)?.path === 'network';
collect.set('isRepoNetworkGraph', [
```

File: index.ts [hunk 2]
```diff
@@ -386,6 +386,16 @@ collect.set('isFileFinder', [
 	'https://github.com/sindresorhus/refined-github/find/master',
 ]);
 
+export const isForksList = (url: URL | HTMLAnchorElement | Location = location): boolean => getRepo(url)?.path === 'network/members';
+collect.set('isForksList', [
```
Comment (fregante) [inline index.ts:390]:
```suggestion
export const isRepoForksList = (url: URL | HTMLAnchorElement | Location = location): boolean => getRepo(url)?.path === 'network/members';
collect.set('isRepoForksList', [
```

**Comment by fregante** [comment]
Thank you! You can run `npm i github-url-detection@latest` in your Refined GitHub PR (in a few minutes)

**Comment by yakov116** [comment]
He has node < 15. the lock will get messed up.

**Comment by fregante** [comment]
Time to update it then, as `contributing.md` suggests

**Comment by payne911** [comment]
I downloaded `Node` a month ago from their official website, and the latest LTS is < 15 : <https://nodejs.org/en/download/>

image

I guess I didn't pay attention to the fact that it was below the required minimum (I updated my version precisely because of that `contributing.md`), assuming the latest LTS would match it. My bad.

**Pull request merged**